src/actions/sources/createPrettySource.js

```diff
@@ -25,19 +47,19 @@ export function createPrettySource(sourceId) {
       frames = await updateFrameLocations(frames, sourceMaps);
     }
 
-    const prettySource = {
-      url,
-      id,
-      isPrettyPrinted: true,
-      text: code,
-      contentType: "text/javascript",
-      frames,
-      loadedState: "loaded"
-    };
+    prettySource.text = code;
+    prettySource.frames = frames;
+    prettySource.loadedState = "loaded";
 
     dispatch({
-      type: "ADD_SOURCE",
-      source: prettySource
+      type: "UPDATE_SOURCES",
+      sources: [
+        {
+          id: sourceId,
+          loadedState: "loaded"
+        },
+        prettySource
+      ]
     });
```

Comment:
we should be updating the pretty source here.

Reply:
It's update above https://github.com/devtools-html/debugger.html/pull/4850/files#diff-69b4b3dbfca379d059dd7360f8c53a52R50